pkg/virt-controller/services/template.go

```diff
@@ -852,6 +853,15 @@ func (t *templateService) RenderHotplugAttachmentPodTemplate(volumes []*v1.Volum
 					},
 					SecurityContext: &k8sv1.SecurityContext{
 						SELinuxOptions: &k8sv1.SELinuxOptions{
+							// FIXME: Forcing an SELinux level without categories is a security risk
+							// This pod will contain a disk image shared with a virt-launcher pod.
+							// If we didn't force a level here, one would be auto-generated with a set of categories
+							// different from the one of its companion virt-launcher. Therefore, SELinux would prevent
+							// virt-launcher('s compute container) from accessing the disk image.
+							// The proper fix here is to force the level of this pod to match the one of virt-launcher.
+							// Unfortunately, pods MCS levels are not exposed by the API. Therefore, we'd have to
```

Comment:
Yes, as we spoke it might make sense to transparently handle this in virt-handler. Something to explore in the future but this is definitely a big step forward.